recipe/meta.yaml

```diff
@@ -44,19 +44,23 @@ requirements:
   run:
     - python
     - {{ pin_compatible('numpy') }}
+    - cython
     - numexpr
+    - blosc
+    - c-blosc2
     - packaging
     - py-cpuinfo
 
 test:
   requires:
     - setuptools
+    - pip
   commands:
     - pt2to3 -h
     - ptdump -h
     - ptrepack -h
     - pttree -h
-
+    - pip check
```

Comment:
I would expect this to fail because `blosc2` (the python package) is not installed.
We have a introduced a patch to be able to do it.

Reply:
If the `pip check` fails for `pytables` it also fails for all packages which depend on `pytables` and all packages that depend on these packages. So accepting `pytables` to fail is very dangerous for the whole `conda-forge` project.

Reply:
OK, in this case yo should also patch `requirements.txt` and remove `cython` and `blosc2`

Reply:
I added an alternative version of this pull request in https://github.com/conda-forge/pytables-feedstock/pull/87 - which updates the `requirements.txt`